gnome-image-installer/pages/install/gis-scribe.c

```diff
@@ -667,8 +667,8 @@ gis_scribe_write_thread_copy (GisScribe     *self,
     {
       g_autoptr(GError) local_error =
         g_error_new (GIS_IMAGE_ERROR, GIS_IMAGE_ERROR_WRONG_SIZE,
-                     "wrote %" G_GUINT64_FORMAT " bytes, "
-                     "expected to write %" G_GUINT64_FORMAT " bytes",
+                     _("Wrote %'" G_GUINT64_FORMAT " bytes, but "
+                       "expected to write %'" G_GUINT64_FORMAT " bytes."),
```

Comment:
Is this what you were playing with `g_format_size_full()` for? Decided against using it in the end?

Reply:
I didn't want to use `g_format_size_full()` because it only supports reporting the exact size in addition to (not instead of) the rounded size, ie “1.2 GB (1,234,567,890 bytes)”. I only want the second part, on the basis that the one time this actually happened on a correctly-signed image, the message would have been (with this change):

> wrote 101,241,834,086 bytes, expected to write 101,241,833,984 bytes

Whereas using `g_format_size_full (..., G_FORMAT_SIZE_LONG_FORMAT)` would give:

> wrote 101.2 GB (101,241,834,086 bytes), expected to write 101.2 GB (101,241,833,984 bytes)

The rounded size provides no useful information here!

(This condition can only happen if there is a bug in our image builder (as there was), because this check is performed after verifying the GPG signature.)

Reply:
(But I did learn the `%'d` syntax from reading the `g_format_size_full()` source.)

Reply:
You may potentially want to put that in a comment by the code, but the need for it is a bit borderline.